packages/paste-core/components/heading/src/PropTypes.ts

```diff
@@ -0,0 +1,15 @@
+import * as PropTypes from 'prop-types';
+import {HeadingVariants, asTags} from './types';
+
+export const HeadingPropTypes = {
+  as: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div', 'label', 'span'] as asTags[]).isRequired,
+  marginBottom: PropTypes.oneOf(['space0']) as PropTypes.Validator<'space0' | null | undefined> | undefined,
```

Comment:
What's with the `as` on this line 0_0 ? In other components this line doesn't need an `as` at all...

Reply:
For what ever reason, if I move this proptype to a different file typescript was inferring it’s type incorrectly. If it’s in the file where  the component is, it doesn’t complain.

I’ll see if it was my VS Code playing up, but all I’m doing is explicitly recasting it to the type it infers correctly when it’s in the component file. Super weird

Reply:
Seems to be working locally now. Must have been a funny 5 minutes with VS Code

Reply:
I lied, had to revert back to this original state for typescript to compile.